CHANGELOG

```diff
@@ -1,3 +1,18 @@
+Version 0.7.1 (2020-06-26)
+--------------------------
+Bump geoip2 to 2.13.1 (#136)
+Bump cats-effect to 1.4.0 (#137)
+Bump cats-core to 1.6.1 (#138)
+Bump scala-lru-map to 0.4.0 (#139)
```

Comment:
This will bring cats 2 and cats-effect 2, which makes above bumps useless.

Reply:
I bumped both cats-effect and cats-core to 2.x.x.